scripts/commands/checkLinks.ts

```diff
@@ -138,13 +151,28 @@ function markdownFromNotebook(source: string): string {
 async function loadFilesAndLinks(
   filePaths: string[],
 ): Promise<[File[], Link[], Link[]]> {
+  const args = readArgs();
+
   const fileList: File[] = [];
   const internalLinkList: Link[] = [];
   const externalLinkList: Link[] = [];
 
   // Auxiliary Map to avoid link duplications
   const linkMap = new Map<string, string[]>();
 
+  // Files inside the folders we want to ignore
+  const filesInFoldersToIgnores = await globby(FOLDERS_TO_IGNORES);
+
+  const releaseNotes = await globby("docs/api/qiskit/release-notes/*");
+  const latestReleaseNote = releaseNotes
```

Comment:
Consider moving this to a helper function since it has a decent bit of implementation details in it. But I'm not convinced that's a good change

Reply:
I like the idea. I moved the code into a helper function called `lastestQiskitReleaseNote`, which can be easily extended to return other release notes from other repos in the future.